### ISSUE TITLE
"type" vs "scopeType" in the YAML config file

### ISSUE THREAD

**Issue by DrMint** [webrecorder/browsertrix-crawler#82]
Hi! So while testing the config files I think I've found a discrepancy in the README.md.
So we are given this example:

```
seeds:
  - url: https://webrecorder.net/
    depth: 1
    type: "prefix"
```

Then when I tried to change type to "none", it didn't change anything. This is confirmed by the logs (using the parameter --logging stats,debug):

```
Seeds [
  ScopedSeed {
    url: 'https://webrecorder.net/',
    include: [ /^https:\/\/webrecorder\.net\// ],
    exclude: [],
    scopeType: 'prefix',
    sitemap: false,
    allowHash: false,
    maxDepth: 99999
  }
]
```

When using "scopeType" instead of "type", it does allow me to change the scope type. So I think it's either a problem with the way the config file is parsed, or an error in the documentation.

**Comment by ikreymer** [comment]
Yes, it has been renamed to `scopeType`, the documentation is being fixed as part of #81

**Comment by ikreymer** [comment]
Merged, Fixed in 0.4.4!

**Comment by DrMint** [comment]
Awesome! Along with the additional example, it's much clearer now.
